docs/generate.js

```diff
@@ -24,6 +24,9 @@ function generate(pathname) {
 		if (pathname.match(/\.md$/)) {
 			var outputFilename = pathname.replace(/\.md$/, ".html")
 			var markdown = fs.readFileSync(pathname, "utf-8")
+			var splitDescription = markdown.split('#### meta-description') // grabs all from here to bottom of page, so must be last
+			markdown = splitDescription[0]
+			var metaDescription = splitDescription[1] ? '<meta name="description" content="' + splitDescription[1] + '">' : '<meta name="nosnippets">'
```

Comment:
You may want to escape `/["<&]/g` to `{"\"": "&quot;", "<": "&lt;", "&": "&amp;"}`.

Reply:
Is this a complete list of characters you'd want escaped, or should I try to be all encompassing?